src/IotHub/IotHub/help/Invoke-AzIotHubManualFailover.md

```diff
@@ -0,0 +1,183 @@
+﻿---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.IotHub.dll-Help.xml
+Module Name: Az.IotHub
+online version: https://docs.microsoft.com/en-us/powershell/module/az.iothub/invoke-aziothubmanualfailover
+schema: 2.0.0
+---
+
+# Invoke-AzIotHubManualFailover
```

Comment:
@markcowl , I suppose `Start` is a better verb for this command, because to trigger a manual failover (then let the iot hub handle itself) sounds more like an *asynchronous* thing. What you you think?